<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tweedia</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
          <widget class="QTableView" name="tableView"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QMdiArea" name="mdiArea"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenDatabase"/>
    <addaction name="actionCreate_ObsobjTable"/>
    <addaction name="actionOpen_Script"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddObsobj"/>
    <addaction name="actionDeleteObsobj"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTweediaWiki"/>
    <addaction name="actionTweediaWebsite"/>
    <addaction name="separator"/>
    <addaction name="actionReportBug"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTweedia">
    <property name="title">
     <string>Tweedia</string>
    </property>
    <addaction name="actionAbout_Tweedia"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionArrange_Game"/>
    <addaction name="actionStart_Game"/>
   </widget>
   <addaction name="menuTweedia"/>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGame"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNamed_pipe">
   <property name="text">
    <string>named pipe</string>
   </property>
  </action>
  <action name="actionLoad_plugin">
   <property name="text">
    <string>load plugin</string>
   </property>
  </action>
  <action name="actionDelete_observation">
   <property name="text">
    <string>delete observation</string>
   </property>
  </action>
  <action name="actionStart_process">
   <property name="text">
    <string>start process</string>
   </property>
  </action>
  <action name="actionRefreash">
   <property name="text">
    <string>refreash</string>
   </property>
  </action>
  <action name="actionAddObsobj">
   <property name="text">
    <string>Add Obsobj</string>
   </property>
  </action>
  <action name="actionDeleteObsobj">
   <property name="text">
    <string>Delete Obsobj</string>
   </property>
  </action>
  <action name="actionRunObsobj">
   <property name="text">
    <string>Run Obsobj</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionInitialize_Database">
   <property name="text">
    <string>Create ObsobjTable</string>
   </property>
  </action>
  <action name="actionOpen_Script">
   <property name="text">
    <string>Open Script</string>
   </property>
  </action>
  <action name="actionCreate_TblObservation">
   <property name="text">
    <string>Create ObservationTable</string>
   </property>
   <property name="toolTip">
    <string>Create ObservationTable</string>
   </property>
  </action>
  <action name="actionCreate_ObsobjTable">
   <property name="text">
    <string>Create ObsobjTable</string>
   </property>
  </action>
  <action name="actionAbout_Tweedia">
   <property name="text">
    <string>About Tweedia</string>
   </property>
  </action>
  <action name="actionTweediaWebsite">
   <property name="text">
    <string>Tweedia Website</string>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="text">
    <string>Report Bug</string>
   </property>
  </action>
  <action name="actionTweediaWiki">
   <property name="text">
    <string>Tweedia Help (Wiki)</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionBuild_Model">
   <property name="text">
    <string>Build Model</string>
   </property>
  </action>
  <action name="actionArrange_Game">
   <property name="text">
    <string>Arrange Game</string>
   </property>
  </action>
  <action name="actionStart_Game">
   <property name="text">
    <string>Start Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
